instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/Servlet3SnippetInjectingResponseWrapper.java

```diff
@@ -153,7 +153,8 @@ public boolean isContentTypeTextHtml() {
     if (contentType == null) {
       contentType = super.getHeader("content-type");
     }
-    return contentType != null && contentType.startsWith("text/html");
+    return contentType != null
+        && (contentType.startsWith("text/html") || "application/xhtml+xml".equals(contentType));
```

Comment:
is this valid `Content-Type`? (wondering if we should do startsWith here)

```
application/xhtml+xml; charset=utf-8
```

Reply:
Yes i agree, this change will have better  compatibility